code/modules/reagents/reagent_containers/food/condiment.dm

```diff
@@ -14,6 +14,16 @@
 	possible_transfer_amounts = list(1,5,10)
 	center_of_mass = list("x"=16, "y"=6)
 	volume = 50
+	var/next_shake = TRUE
```

Comment:
```suggestion
	var/next_shake
```
This shouldn't be set to a boolean since it will be set to a number value in shake()